src/main/kotlin/com/cognifide/gradle/aem/common/AemExtension.kt

```diff
@@ -149,7 +156,9 @@ open class AemExtension(@Internal val project: Project) {
         throw InstanceException("Instance named '$nameMatcher' is not defined.")
     }
 
-    fun filterInstances(nameMatcher: String = props.string("aem.instance.name") ?: "$environment-*"): List<Instance> {
+    fun filterInstances(
+        nameMatcher: String = props.string("aem.instance.name") ?: "$environment-*"
```

Comment:
if detekt does not complain about line length, please do not make such changes if you musn't

Reply:
OK